tests/unit/resource/shared_memory_resource_tests.cpp

```diff
@@ -51,12 +51,12 @@ class SharedMemoryTest : public ::testing::Test {
 
     if (shmem_resource == nullptr) {
       auto& rm = umpire::ResourceManager::getInstance();
-      auto traits{umpire::get_default_resource_traits("SHARED")};
+      auto traits{umpire::get_default_resource_traits("SHARED::POSIX")};
       ASSERT_EQ(traits.scope, umpire::MemoryResourceTraits::shared_scope::node);
       ASSERT_EQ(traits.resource, umpire::MemoryResourceTraits::resource_type::shared);
 
       traits.size = m_segment_size;
-      ASSERT_NO_THROW(allocator = rm.makeResource("SHARED::node_allocator", traits););
+      ASSERT_NO_THROW(allocator = rm.makeResource("SHARED::POSIX::node_allocator", traits););
```

Comment:
This is actually REQUIRED for this to work. For the strategy tests to pass, on ruby, with IPC and MPI3 enabled, it kept trying to use the MPI3 shared memory unless I explicitly asked for `rm.makeResource("SHARED::POSIX::some_name", ....` - is that expected?

Reply:
Comments added - I assume that is expected since I haven't heard otherwise...